### ISSUE TITLE
Simplefetch: WM not displaying correctly.

### ISSUE THREAD

**Issue by mothsarecool524** [Chick2D/neofetch-themes#59]
Using Hyprland, Simplefetch displays "sway" as WM. I don't see anywhere that can be configured.

![image](https://github.com/Chick2D/neofetch-themes/assets/124483129/c5c91099-c3d2-4c0e-bb9c-6cf725e58e5b)

**Comment by Chick2D** [comment]
I got a feeling this might be a neofetch issue

**Comment by Deftera186** [comment]
> I got a feeling this might be a neofetch issue

 Yep, it seems like Neofetch misclassifies Hyprland as a DE and displays the WM as Sway. dylanaraps/neofetch#2366
 
 If you really want, I guess you can just edit this line to manually print "Hyprland". 
 https://github.com/Chick2D/neofetch-themes/blob/cef1e10d702ae5dc5865ffd57e1c52b9143fff2f/small/simplefetch.conf#L10
 
 To:
 ```bash
 prin "$(color 13) " "Hyprland"
 ```
